Cargo.toml

```diff
@@ -99,7 +99,7 @@ derive-new = "0.7.0"
 tracing-subscriber = { version = "0.3", features = ["json", "env-filter"] }
 event-listener = "5.3.1"
 flume = "0.11"
-form_urlencoded = "1.2.1"
+form_urlencoded = "=1.2.1"
```

Comment:
This is not a zenoh internal dependency.

Reply:
Oups... blind search/replace 😅